cmd/virt-launcher/virt-launcher.go

```diff
@@ -610,15 +611,21 @@ func RemoveContents(dir string) error {
 
 func terminateIstioProxy() {
 	if istioProxyPresent() {
-		resp, err := http.Post(fmt.Sprintf("http://localhost:%d/quitquitquit", infraconfigurators.EnvoyMergedPrometheusTelemetryPort), "", nil)
+		httpClient := &http.Client{
```

Comment:
Consider using the same httpClient for both the `GET` and the `POST` - this is in accordance to the general recommendation to reuse http clients as opposed to recreating them, such that the underlying transport can reuse its internal state (cached connection, etc).

Reply:
Done